packages/ai/core/generate-text/smooth-stream.ts

```diff
@@ -1,13 +1,22 @@
-import { InvalidArgumentError } from '@ai-sdk/provider';
 import { delay as originalDelay } from '@ai-sdk/provider-utils';
 import { TextStreamPart } from './stream-text-result';
 import { ToolSet } from './tool-set';
+import { InvalidArgumentError } from '@ai-sdk/provider';
 
 const CHUNKING_REGEXPS = {
-  word: /\s*\S+\s+/m,
-  line: /[^\n]*\n/m,
+  word: /\S+\s+/m,
+  line: /\n+/m,
```

Comment:
this will lead to the error where empty lines get matched and enqued

Reply:
wait will it, this `/[^\n]*\n/m` matches the last newline, but `\n+` with my change will still match the last + insert trailing newlines